docker/jvm/Dockerfile

```diff
@@ -56,8 +56,9 @@ ARG build_date
 LABEL org.label-schema.name="kafka" \
       org.label-schema.description="Apache Kafka" \
       org.label-schema.build-date="${build_date}" \
-      org.label-schema.vcs-url="https://github.com/apache/kafka" \
-      maintainer="Apache Kafka"
```

Comment:
Why are we making this change? If there's a rational for making it, I'd rather do it in a separate PR instead.

Reply:
I should revert it, the MAINTAINER is deprecated
[doc](https://docs.docker.com/reference/build-checks/maintainer-deprecated/)

Reply:
Thanks for the explanation. Let's fix that in another PR.

Reply:
I will address it in this [Jira](https://issues.apache.org/jira/browse/KAFKA-17834)